Lib/fontTools/ttLib/tables/TupleVariation.py

```diff
@@ -446,6 +444,125 @@ def getTupleSize_(flags, axisCount):
 			size += axisCount * 4
 		return size
 
+	def getDeltaType(self):
+		""" Check if deltas are (x, y) as in gvar, or single values as in cvar.
+		Returns a string ("gvar" or "cvar"), or None if empty.
+		"""
+		firstDelta = next((c for c in self.coordinates if c is not None), None)
+		if firstDelta is None:
+			return  # empty or has no impact
```

Comment:
us hasImpact?

Reply:
yes, I could call `hasImpact` here, but doing this saves me an extra for loop and method call